README.md

```diff
@@ -11,25 +11,91 @@ _docs:_
 
 ## Motivation
 
-Clojure's built-in data structures get you a long way when trying to codify and solve difficult problems. A solid selection of core functions allow you to easily create and access core data structures. Unfortunately, this flexibility does not extend to testing: a comprehensive yet extensible way to assert that the data fits a particular structure seems to be lacking.
+Clojure's built-in data structures get you a long way when trying to codify and solve difficult problems. A solid selection of core functions allow you to easily create and access core data structures. Unfortunately, this flexibility does not extend to testing: we seem to be missinng a comprehensive yet extensible way to assert that the data fits a particular structure.
 
-This library addresses this issue by providing composable matcher combinators that can be used as building blocks to effectively test functions that evaluate to nested data-structures.
+This library addresses this issue by providing composable matcher combinators that can be used as building blocks to test functions that evaluate to nested data-structures more effectively.
 
 ## Features
 
+- Matchers for scalar and structural values
+  - Good readability supported by default interpretations of Clojure types as matchers
 - Pretty-printed diffs when the actual result doesn't match the expected matcher
-- Integrates with `clojure.test` and `midje`
-- Good readability by providing default interpretations of core Clojure data-structures as matcher combinators
+- Integration with `clojure.test` and `midje`
 
-| Midje checkers | Matcher combinators |
-| ------- | ----- |
-| ![midje checkers](doc/images/midje_check.png) | ![matcher combinators check](doc/images/matcher_check.png) |
```

Comment:
we could ✂️ those files too

Reply:
very happy to see this horrible screenshots go away :)